src/screens/ENSSearchSheet.js

```diff
@@ -61,7 +61,6 @@ export default function ENSSearchSheet() {
   } = useENSRegistrationCosts({
     name: debouncedSearchQuery,
     rentPrice: registrationData?.rentPrice,
-    sendReverseRecord: true,
```

Comment:
we only need this one in the confirm registration screen, passing fixed `true` was causing differences in the different places we're calling this hook